prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -457,15 +457,7 @@ workflow_has_parameter AOD && add_W o2-aod-producer-workflow "--info-sources $AO
 
 # ---------------------------------------------------------------------------------------------------------------------
 # Quality Control
-if [[ ! -z $QC_JSON_FROM_OUTSIDE ]]; then
-    if [[ -z $QC_CONFIG_PARAM ]]; then
-        QC_CONFIG_PARAM="--local --host $QC_HOST"
-    fi
-    echo "qc json from outside $QC_JSON_FROM_OUTSIDE"
-    workflow_has_parameter QC && add_W o2-qc "--config json://$QC_JSON_FROM_OUTSIDE $QC_CONFIG_PARAM" "" 0
-else
-    workflow_has_parameter QC && { source $O2DATAPROCESSING_ROOT/production/qc-workflow.sh; [[ $? != 0 ]] && exit 1; }
-fi
+workflow_has_parameter QC && { source $O2DATAPROCESSING_ROOT/production/qc-workflow.sh; [[ $? != 0 ]] && exit 1; }
```

Comment:
Good, for reference, see https://github.com/AliceO2Group/O2DataProcessing/commit/79be7b40377396a11f451774e71cf8adbf6a4d56